ee/be/modules/350-node-local-dns/monitoring/grafana-dashboards/kubernetes-cluster/dns-node-local.json

```
@@ -6549,6 +3611,7 @@
   },
   "timezone": "",
   "title": "DNS (node local)",
-  "uid": "HrJTjvuiz",
-  "version": 3
-}
+  "uid": "HrJTjvuiz2",
```

It's not good idea to change the dashboard uid.
```suggestion
  "uid": "HrJTjvuiz",
```